src/BigNumber.php

```diff
@@ -101,6 +101,17 @@ public static function of($value) : BigNumber
         return new BigInteger($integral);
     }
 
+    /**
+     * @param mixed $value
+     * @return string
+     */
+    private static function convertToString($value): string
```

Comment:
This can be inlined in `of()` for performance reasons, the code is short enough to not justify an extra method.